added request reviewer button functionality

DamiOgboja opened the pull request:
# Proposed Changes

* added the request review api and button and then added it to the company page

Fixes # .

DamiOgboja committed:
added request reviewer button functionality

added the request review api and button and then added it to the company page

DamiOgboja committed:
prettier formatting

DamiOgboja committed:
formatting

DamiOgboja committed:
formatting python

DamiOgboja committed:
more...formatting

does it really have to be alphabetical? who made this and why?

DamiOgboja committed:
one more time!

DamiOgboja commented:
if one of yall can fix the isort thing thatd be great, i give up. i dont even underdstand y its upset or why we need to use this isort thing if its so annoying and im tired.

JimtheBurger committed:
Merge remote-tracking branch 'origin/development' into 135-add-review-request-button

JimtheBurger committed:
fix isort

JimtheBurger reviewed (changes requested):
It looks like you cannot Request a Review of a company that isn't in the database already. If you make it so that if the company doesn't exist and a review is requested it is added to the database that would be great.

DamiOgboja committed:
fixed constraint error

just reorganized the ordering of the process, made it check if company existed in the table and added it if not BEFORE the rest of the process

DamiOgboja committed:
Merge branch '135-add-review-request-button' of https://github.com/AdverseApps/Adversarial-Apps-Web into 135-add-review-request-button

DamiOgboja committed:
console log addition

DamiOgboja committed:
Update package-lock.json

DamiOgboja committed:
formatting

DamiOgboja committed:
Merge branch 'development' into 135-add-review-request-button
